docs/reference/ml/ml-shared.asciidoc

```diff
@@ -458,6 +458,18 @@ that moment in time. See
 This check runs only on real-time {dfeeds}.
 end::delayed-data-check-config[]
 
+tag::delayed-data-check-config-check-frequency[]
+The how often should the {dfeed} search for late data.
+It must be smaller than the delayed data `check_window`.
+Additionally, if {dfeed} frequency is set, `check_frequency` must
+be larger than `frequency`.
+Delayed data checks cannot occur more often than the configured `frequency`.
+But, setting `check_frequency` higher will increase the time between delayed
+data checks.
+
```

Comment:
This final paragraph is not being formatted properly in the output. It either needs to be linked to the previous paragraph with a "+" or else just merged into the previous paragraph (I think the latter's my preference)
```suggestion
+
```